stories/voice-search.stories.ts

```diff
@@ -0,0 +1,186 @@
+import { storiesOf } from '@storybook/html';
+import { withHits } from '../.storybook/decorators';
+
+storiesOf('VoiceSearch', module)
+  .add(
+    'default',
+    withHits(({ search, container, instantsearch }) => {
+      const descContainer = document.createElement('div');
+      const realContainer = document.createElement('div');
+      container.appendChild(descContainer);
+      container.appendChild(realContainer);
+      descContainer.innerHTML = `
+        <p>To see this button disabled, test it on unsupported browsers like Safari, Firefox, etc.</p>
+      `;
+
+      search.addWidget(
+        instantsearch.widgets.voiceSearch({
```

Comment:
Until we have a proper definition for `widgets` we should use the source of the widget. Otherwise we don't leverage TypeScript to warn us about the usage of the widget. Here the usage does not match the definition of the types since the widget is inferred as `any`. cc @francoischalifour